_posts/2019-04-09-sr2019-tlc-crowned-champion.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: news
+title: The Ladies' College Wins Student Robotics 2019
+---
+
+{% include figure.html
+           src="/images/content/news/sr2019/sr2019-photo.jpg"
+           caption="All the teams from SR2019" %}
+
+
+That's a wrap on Student Robotics 2019! Our teams have been working hard since November to design, build, and test their robots; finally seeing the fruits of their work on the competition weekend. It was great to see our competitors quickly adapting their robots, from the physical design to the software, to overcome unforeseen challenges; giving themselves every small advantage towards claiming victory.
+
+The Challenge: Caldera
+----------------------
+
+{% include figure.html
+           class="right"
+           src="/images/content/news/sr2019/arena.jpg"
+           caption="The arena with a raised volcano area" %}
+
+{% include figure.html
+           class="right"
+           src="/images/content/news/sr2019/tlc-launcher.jpg"
+           caption="The Ladies' College's token launcher" %}
+
+           
+{% include figure.html
+           class="right"
+           src="/images/content/news/sr2019/swi.jpg"
+           caption="South Wilts Grammar School's volcano-climbing robot in the Caldera" %}
+
+Our game this year, Caldera, challenged teams to control scoring zones with tokens while navigating an arena with raised platforms. Twenty-five scoring zones were arranged in a grid, varying in value from two points for each of the outermost 16 zones, a ring of eight on a raised “volcano” worth seven points apiece, to the the central “caldera” which was worth thirty points. Parking your robot in a zone tripled the points of that zone and the 4 adjacent zones.
+
+There were a range of inventive designs this year. Many teams opted for tank tracks as a method of climbing the volcano. Instead of climbing the volcano, a few teams built catapult mechanisms with varying degrees of success. Others scooped up tokens and deposited them on the low scoring areas and/or the edge of the volcano. Some teams lifted tokens onto top-mounted conveyor belts which dropped them onto the volcano's surface.
+
+Some of the more notable designs were:
+
+- [The Ladies' College](http://www.ladiescollege.com/) with their consistent, successful catapult.
```

Comment:
Would these links make more sense as links to images of the robots (or something specific to the team)? As it stands, I'm not sure the links add a lot of value here.

Reply:
Planning to do this when we have a central place for photos.

Reply:
ok, for info we do already have images of the robots which we can use (they're at least in the compstate, though possibly elsewhere too).

Reply:
I don't know about others, but I don't think I have a picture of CRB or a decent one of HAB

Reply:
https://github.com/srobo/sr2019-comp/tree/master/teams/images has the images of the robots.
There should be the images from the team photos booth somewhere too.